README.md

```diff
@@ -1 +1,12 @@
-# react-repo, All pesonal react project during the GYM Rwanda training.
+# React-repo 
+
+ This repo contains all exercises from React Curriculum from the GYM Rwanda.
+ All exercices are in pages folder and related components are in Components
+
+## Run the Project
+
+1) npm install or yarn install
```

Comment:
As you don't have yarn.lock file that can allow package installation with yarn, `yarn install` should be removed as the project installation command, unless you add the yarn.lock fike.

Reply:
Yes, I will work on  it.